src/apps/material/material.dev.tsx

```diff
@@ -790,7 +790,7 @@ export default {
       name: "Instant loan config",
       control: { type: "text" },
       defaultValue:
-        '{\n        "threshold": "1",\n        "matchString": "14 dages lån - bogligt (kan ikke reserveres)",\n        "enabled": "true"      }'
+        '{\n        "threshold": "1",\n        "matchStrings": ["31 dages lånetid til alm lånere"],\n        "enabled": "true"      }'
```

Comment:
I do not understand the necessity of this change and the related changes in our fixtures.

As I see it "14 dages lån - bogligt (kan ikke reserveres)" is actually a real world sample of a group name that should trigger instant loans. The change seems to update it to something where that is no longer the case.

We update it in the setup as well as in the fixture so could we avoid this part of the PR entirely?

Reply:
I don't know exactly how the matchString setting and the fixture got out of sync. But I have updated the fixture with latest work holding data. And I have pointed the branch material assertion at "Hovedbiblioteket" to emphasise the relationship between branches, material descriptions and material available status.